ibexa/ckeditor-premium/4.6/manifest.json

```
@@ -0,0 +1,14 @@
+{
+  "aliases": [],
+  "bundles": {
+    "Ibexa\\Bundle\\CkeditorPremium\\IbexaCkeditorPremiumBundle": ["all"]
```

```suggestion
    "Ibexa\\Bundle\\CKEditorPremium\\IbexaCKEditorPremiumBundle": ["all"]
```

This is correct capitalization.